### PR TITLE
Autotests: #3008 Write autotest for Copy/Cut/Paste Actions

### PR DESCRIPTION
**Pull request by Zhirnoff**
- planning add 15 tests for 'copy-cut-paste';

## How the feature works? / How did you fix the issue?
(Screenshots, videos, or GIFs, if applicable)

## Check list
- [ ] unit-tests written
- [x] e2e-tests written
- [x] documentation updated
- [x] PR name follows the pattern `#1234 – issue name`
- [x] branch name doesn't contain '#'
- [x] PR is linked with the issue
- [x] base branch (master or release/xx) is correct
- [x] task status changed to "Code review"

### CODE REVIEW

**Review by MartaWilliams — APPROVED** [review]

File: ketcher-autotests/tests/User-Interface/Editing-Tools/Copy-Cut-Paste/copy-cut-paste.spec.ts [hunk 1]
```diff
@@ -285,6 +285,301 @@ test.describe('Copy/Cut/Paste Actions', () => {
...
+      /*
+    Test case: EPMLSOPKET-1719
+    Description: The correct structure is pasted on the canvas.
+    All Generic S-Group are correctly rendered.
+    User is able to edit the pasted structure.
+    */
+      await openFileAndAddToCanvas('query-feat.mol', page);
+      await cutAndPaste(page);
+      await page.mouse.click(CANVAS_CLICK_X, CANVAS_CLICK_Y);
+      await delay(DELAY_IN_SECONDS.TWO);
```
Comment (Nitvex) [inline ketcher-autotests/tests/User-Interface/Editing-Tools/Copy-Cut-Paste/copy-cut-paste.spec.ts:351]: Why do we need this delay?

File: ketcher-autotests/tests/User-Interface/Editing-Tools/Copy-Cut-Paste/copy-cut-paste.spec.ts [hunk 2]
```diff
@@ -285,6 +285,301 @@ test.describe('Copy/Cut/Paste Actions', () => {
...
+    await selectAtomInToolbar(AtomButton.Nitrogen, page);
+    await clickOnAtom(page, 'C', 12);
+  });
+
+  test('Copy and paste reaction with changed arrow', async ({ page }) => {
+    /*
+    Test case: EPMLSOPKET-2873
+    Description: Copied reaction has Failed Arrow with default size and position.
+    */
+    const x = 500;
```
Comment (Nitvex) [inline ketcher-autotests/tests/User-Interface/Editing-Tools/Copy-Cut-Paste/copy-cut-paste.spec.ts:504]: I propose to make these constants test-wide (move to upper scope).

File: ketcher-autotests/tests/User-Interface/Editing-Tools/Copy-Cut-Paste/copy-cut-paste.spec.ts [hunk 3]
```diff
@@ -285,6 +285,301 @@ test.describe('Copy/Cut/Paste Actions', () => {
...
+    await page.mouse.click(x, y);
+  });
+
+  test('Cut and Paste reaction by hotkeys and edit', async ({ page }) => {
+    /*
+    Test case: EPMLSOPKET-1730
+    Description: Copied objects are pasted correctly.
+    The structure is cut (and then is pasted) with reaction arrow and plus sign
+    */
+    await openFileAndAddToCanvas('reaction.rxn', page);
```
Comment (Nitvex) [inline ketcher-autotests/tests/User-Interface/Editing-Tools/Copy-Cut-Paste/copy-cut-paste.spec.ts:491]: May be it is worth creating a separate function? 
You're repeating all the tests, but the only structure changes

**Review by nanoblit — APPROVED** [review]
I agree with the points made by Nitvex, but I don't see anything else that looks wrong.

**Pull request merged**